client/analytics/filters/index.js

```diff
@@ -0,0 +1,84 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
+import { Fragment } from '@wordpress/element';
+import { noop } from 'lodash';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import AdvancedFilters from 'components/advanced-filters';
+import Card from 'components/card';
+import DatePicker from 'components/date-picker';
+import FilterPicker from 'components/filter-picker';
+import { H, Section } from 'layout/section';
+import './style.scss';
+
+const ReportFilters = ( { advancedConfig, filters, filterPaths, query, path } ) => {
+	let advancedCard = false;
+	switch ( query.filter ) {
+		case 'compare':
+			advancedCard = (
+				<Card
+					title={ __( 'Compare Products', 'wc-admin' ) }
+					className="woocommerce-analytics__compare"
+				>
+					<div className="woocommerce-analytics__compare-body">
+						<input type="search" />
+						<div>Tokens</div>
+					</div>
+					<div className="woocommerce-analytics__compare-footer">
+						<Button onClick={ noop } isDefault>
+							{ __( 'Compare', 'wc-admin' ) }
+						</Button>
+					</div>
+				</Card>
+			);
+			break;
+		case 'advanced':
+			advancedCard = (
+				<AdvancedFilters config={ advancedConfig } filterTitle={ __( 'Orders', 'wc-admin' ) } />
+			);
```

Comment:
The above is partially placeholder, but it does bring up the same label issues that `AdvancedFilters` has, where we want to be able to pass around labels as simply as possible. See #285